engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -43,6 +42,7 @@ object SlickDataAccess {
   private lazy val rootDatabaseConfig = rootConfig.getConfig("database")
   private lazy val databaseConfigName = rootDatabaseConfig.getStringOption("config")
   lazy val defaultDatabaseConfig = databaseConfigName.map(getDatabaseConfig).getOrElse(rootDatabaseConfig)
+  val actorSystem = ActorSystem("slick-retry-scheduler")
```

Comment:
Right, that singleton was a temporary hack for something which no longer matters and we shouldn't continue to feed it. Also agree that there shouldn't be a new system created here